README.md

```diff
@@ -33,6 +33,7 @@ docker pull yanetplatform/builder
 Add alias for run commands on docker:
 ```
 alias yanet-builder="docker run --rm -it -v /run/yanet:/run/yanet -v \$PWD:/project yanetplatform/builder"
+(for Mac: alias yanet-builder="docker run --platform linux/amd64 --rm -it -v /run/yanet:/run/yanet -v \$PWD:/project yanetplatform/builder")
```

Comment:
This is not relevant change, please create another pull request

Reply:
Yes, this is our overall politic to split non-relevant changes even if changes are small.

Reply:
Ok